adept/_vlasov1d/helpers.py

```diff
@@ -244,6 +244,17 @@ def post_process(result: Solution, cfg: Dict, td: str, args: Dict):
 
     f_xr = store_f(cfg, result.ts, td, result.ys)
 
+    diags_dict = {}
+    for k in ["diag-vlasov-dfdt", "diag-fp-dfdt"]:
```

Comment:
I still think it's worth making this list of two strings a well-known constant in some way, either as a yaml list in `cfg["diagnostics"]` or just a python module-level constant like `DFDT_DIAGNOSTICS`.

Reply:
this probably applies to all configuration parameters in some way right? maybe we should leave it for a future, broader refactor